src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReportingService.kt

```diff
@@ -27,12 +28,12 @@ class ReportingService(
     asyncJobLauncher.run(
       performanceReportJob,
       JobParametersBuilder()
-        .addString("user.id", user.id)
         .addString("user.firstName", userDetail.firstName)
         .addString("user.email", userDetail.email)
         .addDate("from", batchUtils.parseLocalDateToDate(from))
         .addDate("to", batchUtils.parseLocalDateToDate(to.plusDays(1))) // 'to' is inclusive
```

Comment:
> 'to' is inclusive

_nit_ renaming the argument to `toIncluding` would make this comment unnecessary and verifiable

Reply:
I think `to` being inclusive would be the expected behaviour - I don't think there would be confustion on the variable - I think it's mainly comment to explain why a day has been added (because converting from localDate to date you need to add 24 hours)